build.gradle

```diff
@@ -18,6 +18,19 @@ dependencies {
     testCompile 'junit:junit:4.12'
 }
 
+sourceSets {
+
+    test {
```

Comment:
I needed to add

```
    main {
        java {
            srcDirs = ['src/main/java']
            exclude 'resources/'
        }
    }
```

to make IntelliJ happy. But it builds without it on the command line.

Reply:
OK - I haven't had this issue with eclipse, but good to know cheers